lib/domain/managers/shared_preferences_manager.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:async';
+
+import 'package:vost/data/sharedpreferences/vost_shared_preferences.dart';
+
+
+/// Class for managing data from Shared Preferences
+/// 
+/// In this class we will manipulate data so that it can be presented to
+/// the UI, for example, convert an int to a DateTime or assigning default
+/// values to Strings
+class SharedPreferencesManager {
+  VostSharedPreferences _sharedPreferences;
+
+  SharedPreferencesManager(this._sharedPreferences);
+
+  //region Test
```

Comment:
I’d use here the `@visibleForTesting` annotation instead.

Reply:
This is not for testing purposes, as in unit testing, but each pair of read/write will be in its own region.

Since there was no key, the first key was "Test", hence the `region Test`